connections/ethereum/connection.go

```diff
@@ -71,7 +75,7 @@ func (c *Connection) Connect() error {
 	c.conn = ethclient.NewClient(rpcClient)
 
 	// Construct tx opts, call opts, and nonce mechanism
-	opts, _, err := c.newTransactOpts(big.NewInt(0), c.gasLimit, c.maxGasPrice)
+	opts, _, err := c.newTransactOpts(big.NewInt(0), c.gasFeeCap, c.gasTipCap)
```

Comment:
I think we still need gas_limit in EIP-1559. 
gasFeeCap + gasTipCap just replace old maxGasPrice.

Reply:
For your reference, https://eips.ethereum.org/EIPS/eip-1559#specification

Reply:
Yup, Updated to use `gasLimit` also.